acsoo/makepot_script

```diff
@@ -38,7 +41,15 @@ if module:
                 pattern, '', file_content, flags=re.MULTILINE)
         pot_file.write(bytes(file_content, 'utf-8'))
     FNULL = open(os.devnull, 'w')
-    for lang in languages:
+    module_languages = languages or []
+    if not module_languages:
+        for filename in os.listdir(i18n_path):
+            is_po_file = filename.endswith(PO_FILE_EXT)
+            if not is_po_file or not isfile(join(i18n_path, filename)):
+                continue
+            language = filename.replace(PO_FILE_EXT, '')
+            module_languages.append(language)
```

Comment:
add instead of append

Reply:
ah, here we have a problem with sublanguage (eg fr_BE) on which we may not want to do the msgmerge, because they contain only overrides on the main languages.

Reply:
At the moment, I think the only way to avoid it is to check if there is an underscore in the filename, then we should consider it's a extension language?